.pipelines/stages/jobs/steps/utils/download-ort.yml

```diff
@@ -5,60 +5,58 @@ parameters:
-      userRepository: 'microsoft/onnxruntime'
-      defaultVersionType: 'specificTag'
-      version: 'v$(ort_version)'
-      itemPattern: '$(ort_filename).${{ parameters.archiveType }}'
-      downloadPath: '$(Build.Repository.LocalPath)'
-    displayName: Download $(ort_filename)
+- task: DownloadPackage@1
+  inputs:
+    packageType: 'nuget'
+    feed: '2692857e-05ef-43b4-ba9c-ccf1c22c437c/7982ae20-ed19-4a35-a362-a96ac99897b7'
+    definition: '$(ort_filename)' # Can also be package name
+    version: '$(ort_version)'
+    extract: false
+    downloadPath: '$(Build.Repository.LocalPath)'
+  displayName: Download Onnxruntime file
+- ${{ if eq(parameters.archiveType, 'zip') }}:
   - task: ExtractFiles@1
     inputs:
-      archiveFilePatterns: '$(Build.Repository.LocalPath)/$(ort_filename).${{ parameters.archiveType }}'
-      destinationFolder: '$(Build.Repository.LocalPath)'
+      archiveFilePatterns: '$(Build.Repository.LocalPath)/*.nupkg'
+      destinationFolder: '$(Build.Repository.LocalPath)/ort'
       cleanDestinationFolder: false
       overwriteExistingFiles: true
     displayName: Unzip OnnxRuntime
   - task: CopyFiles@2
     inputs:
-      SourceFolder: '$(Build.Repository.LocalPath)/$(ort_filename)'
-      TargetFolder: '$(Build.Repository.LocalPath)/ort'
-    displayName: Copy OnnxRuntime to ort
+      SourceFolder: '$(Build.Repository.LocalPath)/ort/runtimes/$(os)-$(arch)/native'
+      TargetFolder: '$(Build.Repository.LocalPath)/ort/lib'
 - ${{ else }}:
-  - task: DownloadPackage@1
-    inputs:
-      packageType: 'nuget'
-      feed: '2692857e-05ef-43b4-ba9c-ccf1c22c437c/7982ae20-ed19-4a35-a362-a96ac99897b7'
-      definition: 'Microsoft.ML.OnnxRuntime.DirectML' # Can also be package name
-      version: '$(ort_version)'
-      extract: false
-      downloadPath: '$(Build.Repository.LocalPath)'
-    displayName: Download $(ort_filename)
+  - script: |
+      mv $(Build.Repository.LocalPath)/$(ort_filename).nupkg $(Build.Repository.LocalPath)/$(ort_filename).zip
+    displayName: Rename OnnxRuntime nuget package to zip
   - task: ExtractFiles@1
     inputs:
-      archiveFilePatterns: '$(Build.Repository.LocalPath)/*.nupkg'
+      archiveFilePatterns: '$(Build.Repository.LocalPath)/$(ort_filename).zip'
       destinationFolder: '$(Build.Repository.LocalPath)/ort'
       cleanDestinationFolder: false
       overwriteExistingFiles: true
     displayName: Unzip OnnxRuntime
   - task: CopyFiles@2
     inputs:
-      SourceFolder: '$(Build.Repository.LocalPath)/ort/runtimes/win-x64/native'
+      SourceFolder: '$(Build.Repository.LocalPath)/ort/runtimes/$(os)-$(arch)/native'
       TargetFolder: '$(Build.Repository.LocalPath)/ort/lib'
-  - task: CopyFiles@2
-    inputs:
-      SourceFolder: '$(Build.Repository.LocalPath)/ort/build/native/include'
-      TargetFolder: '$(Build.Repository.LocalPath)/ort/include'
+  # TODO: Find out why do we need to to have libonnxruntime.so.ort_stable_version
+  - script: |
+      set -e -x
```

Comment:
libonnxruntime.so is a symbolic link to libonnxruntime.so.$ort_stable_version . Why this cp command is needed? What get changed?

Reply:
I was looking for libonnxruntime.so.$ort_version which is not part of the nuget pkg, but it was part of the zip.

Reply:
It might be related to https://github.com/NuGet/Home/issues/12136